proto/backend.proto

```diff
@@ -359,20 +356,6 @@ message AdmissionRequest {
   bytes old_object_bytes = 5;
 }
 
-// ConversionRequest supports converting an object from one version to another 
-message ConversionRequest {
-  // NOTE: this may not include app or datasource instance settings depending on the request
-  PluginContext pluginContext = 1;
-
-  // The object kind
-  GroupVersionKind kind = 2;
```

Comment:
it looks like the new conversion request does not specify the source GVK -- is that accurate?  Does that mean we need to parse the raw bytes to detect it first?  Does k8s know the value before sending?

Reply:
@ryantxu the version can be different for every object in the request. maybe a helper could turn this back into a function that handles one object at a time?